.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
```

Comment:
There are around 260 warnings with this solution are we going to address these now or later?

Reply:
Also can't see the settings template file.

Reply:
Fixed most of the warnings.  Some have been ignored

Reply:
added settings file.